assets/component-rating.css

```diff
@@ -0,0 +1,78 @@
+.rating {
+  display: inline-block;
+  margin: 0;
+}
+
+.rating-star.color-icon-text {
+  --color-icon: rgb(var(--color-base-text));
```

Comment:
Since the `.color-icon-XX` is determined by a theme setting, it would make sense to not limit the scope to only `.rating-star` (tho i am not sure where else we are planning to use this setting). We could simplify and move to `base.css`

```css
.color-icon-text {
  --color-icon: rgb(var(--color-base-text));
}

.color-icon-accent-1 {
  --color-icon: rgb(var(--color-base-accent-1));
}

.color-icon-accent-2 {
  --color-icon: rgb(var(--color-base-accent-2));
}

.color-icon-outline-button {
  --color-icon: rgb(var(--color-base-outline-button-labels));
}
```

Reply:
The reason that I kept it under there is because it isnt being used anywhere else 🤔 But this can def be reused in the future. Moved it 👍